### PR TITLE
Refactor/charts

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207089662029488

### CODE REVIEW

File: src/lib/components/charts/stories/charts.data.ts [hunk 1]
```diff
@@ -1,32 +1,32 @@
 export const CHARTS_DATA = [
```
Comment (MarioBatalha) [inline src/lib/components/charts/stories/charts.data.ts:1]: Is missing types for this data

**Review by MarioBatalha — COMMENTED** [review]
Charts doesn't look good in dark theme, except the first one.

**Review by MarioBatalha — APPROVED** [review]

**Review by KipandaJr — CHANGES_REQUESTED** [review]
Line Chart Expected 
image

Line Chart that we have 
image

File: src/lib/components/charts/charts.types.ts [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
 export interface BaseChartProps {
-  data: any;
-  height?: number | string;
   width?: number | string;
+  height?: number | string;
+  data: Array<DataChartProps>;
+}
+
+export interface BaseAreaLineChartProps extends BaseChartProps {
+  lineType?: 'monotone';
```
Comment (KipandaJr) [inline src/lib/components/charts/charts.types.ts:16]: dosent make sense this should be `lineType?: 'monotone' | <alternative value>`

**Review by KipandaJr — CHANGES_REQUESTED** [review]

**Review by KipandaJr — APPROVED** [review]

**Pull request merged**